lib/policies/oauth2-introspect/oauth2-introspect.js

```diff
@@ -7,7 +7,7 @@ module.exports = function (actionParams) {
   const scannedTokens = [];
   const tokenIntrospection = tokenIntrospectionGenerator(actionParams);
 
-  passport.use('bearer-introspect', new BearerStrategy({ passReqToCallback: true }, (req, accessToken, done) => {
+  passport.use(`bearer-introspect_${actionParams.endpoint}`, new BearerStrategy({ passReqToCallback: true }, (req, accessToken, done) => {
```

Comment:
For consistency, we might want to use the same we did for the `jwt` policy

`const strategyName = `jwt-${uuid()}`;`